lib/message/twitch-types/usernotice.ts

```diff
@@ -52,6 +53,10 @@ export interface EventParams {
   [key: string]: string | number | boolean;
 }
 
+export interface EventParamsMaybe {
+  [key: string]: string | undefined;
```

Comment:
```suggestion
  [key: string]?: string | undefined;
```
I don't think it really matters much here though